sdk/python/kfmd/metadata.py

```diff
@@ -82,21 +86,32 @@ def log(self, artifact):
     This method expects `artifact` to have
       - ARTIFACT_TYPE_NAME stirng field the form of
```

Comment:
nit: stirng to string

Reply:
good catch! I will correct it in subsequence pr to avoid needing a lgtm label again.